connect/build.boot

```diff
@@ -0,0 +1,52 @@
+(set-env!
+ :resource-paths #{"resources"}
+ :dependencies '[[cljsjs/boot-cljsjs "0.9.0" :scope "test"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all]
+         '[boot.core :as boot]
+         '[boot.tmpdir :as tmpd]
+         '[clojure.java.io :as io]
+         '[boot.util :refer [sh]])
+
+(def +lib-version+ "0.6.6")
+(def +version+ (str +lib-version+ "-0"))
+
+(task-options!
+ pom {:project     'cljsjs/connect
+      :version     +version+
+      :description "Library that makes it easy to interact with Relayers that conform to the Standard Relayer API"
+      :url         "https://0xproject.com/"
+      :scm         {:url "https://github.com/0xProject/0x-monorepo/"}})
+
+(defn get-file [fileset file-name]
+  (io/file
+   (:dir (first (filter #(= (:path %) file-name) (boot/user-files fileset))))
+   file-name))
+
+(deftask build-connect []
+  (let [tmp (boot/tmp-dir!)]
+    (with-pre-wrap fileset
+      (doseq [f (->> fileset boot/input-files)
+              :let [target (io/file tmp (tmpd/path f))]]
+        (io/make-parents target)
+        (io/copy (tmpd/file f) target))
+      (doseq [f ["package.json" "webpack.bundle.config.js"]]
+        (io/copy (get-file fileset f) (io/file tmp f)))
+      (binding [boot.util/*sh-dir* (str (io/file tmp))]
+        (do ((sh "npm" "install" "--include-dev"))
+            ((sh "npm" "run" "bundle"))
+            ((sh "npm" "run" "generate-extern"))
+            ((sh "rm" "-rf" "./node_modules"))))
+      (-> fileset (boot/add-resource tmp) boot/commit!))))
+
+(deftask package []
+  (comp
+   (build-connect)
```

Comment:
You could use `run-commands` task here to run those npm commands: https://github.com/cljsjs/boot-cljsjs/blob/master/src/cljsjs/boot_cljsjs/packaging.clj#L371 It will take care of setting up workding directory etc.